src/context_store/dashboard/api_server.py

```
@@ -47,7 +47,28 @@ async def _lifespan(app: FastAPI) -> AsyncIterator[None]:
         app.state.cache = cache
-        app.state.service = DashboardService(storage=storage, graph=graph)
+        retrieval_pipeline = None
+        try:
+            from context_store.retrieval.pipeline import RetrievalPipeline
+
+            retrieval_pipeline = RetrievalPipeline.create_for_dashboard(
+                storage=storage,
+                graph=graph,
+                settings=settings,
+            )
+        except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)